driver/src/main/scala/api/commands/AtlasSearchAggregation.scala

```diff
@@ -33,18 +33,24 @@ private[commands] trait AtlasSearchAggregation[P <: SerializationPack] {
     override def toString = s"AtlasSearch(${operator.toString})"
   }
 
-  /** '''EXPERIMENTAL:''' One or at least one string */
+  /** '''EXPERIMENTAL:''' One or at least one string with optional alternate analyzer specified in multi field*/
```

Comment:
Issue format, please use scalariformFormat task

Reply:
did this, but looks like it didn't change this line

Reply:
Ensure yas suggested, multiline is required for readability.ou format with scalariform, not with settings specific to your IDE.
There as suggested multiline is required for readability.